support/Pipelines/Bbh/ControlId.py

```diff
@@ -31,11 +31,22 @@
     "DimensionlessSpinB": "conformal_spin_b",
     "CenterOfMass": "center_of_mass_offset",
     "LinearMomentum": "linear_velocity",
+    "TotalEnergy": "radial_expansion_velocity",
```

Comment:
Maybe name `AdmMass`, `AdmLinearMomentum` and `AdmAngularMomentumZ` to make clear what quantities are being controlled?